app/Http/Controllers/Front/CartController.php

```diff
@@ -251,4 +259,12 @@ public function storeOrder(CartCheckoutRequest $order){
     public function thankYou(Order $order){
         return view('front.cart.thankyou')->with(['order'=> $order]);
     }
+
+    public function getWeights(){
+        $cartItems=$this->cartRepo->getCartItems()->pluck('options')->pluck('weight');
```

Comment:
Takto by to mozno tiez islo

```suggestion
        return $this->cartRepo->getCartItems()->map(function ($item){
            return $item['qty'] * $item['options']['weight'];
        });
```